shared/common-adapters/popup-menu.desktop.js

```diff
@@ -51,6 +53,10 @@ class Menu extends Component<void, Props, void> {
   }
 }
 
+PopupMenu.defaultProps = {
+  visible: true,
```

Comment:
hmm. maybe visible is used to control animation? if not we should just remove it and use && <PopupMenu>  everywhere